pkgs/development/libraries/libffi/default.nix

```diff
@@ -22,6 +22,9 @@ stdenv.mkDerivation (finalAttrs: {
   # cgit) that are needed here should be included directly in Nixpkgs as
   # files.
   patches = [
+    # https://github.com/libffi/libffi/pull/857
+    # function label needs to come before .cfi_startproc
+    ./label-before-cfi_startproc.patch
```

Comment:
This could be a `fetchpatch` of https://github.com/libffi/libffi/commit/8308bed5b2423878aa20d7884a99cf2e30b8daf7.patch, since it was merged and will therefore live as long as the original repository. (Though this is perfectly fine too.)

Reply:
Whoops, sorry… since most of the comment was omitted I didn’t even notice it.